tests/sentry/api/endpoints/test_organization_codeowners.py

```diff
@@ -0,0 +1,147 @@
+    "raw": "* @MeredithAnya\n",
+}
+
+
+class OrganizationCodeOwnersEndpointTest(APITestCase):
+    method = "GET"
+    endpoint = "sentry-api-0-organization-codeowners"
+
+    def setUp(self):
+        self.owner = self.create_user("motto@life.com")
+        self.user_1 = self.create_user("walter.mitty@life.com")
+        self.user_2 = self.create_user("exec@life.com")
+        self.organization = self.create_organization(name="Life", owner=self.owner)
+        self.team_1 = self.create_team(
+            organization=self.organization,
+            slug="negative-assets",
+            members=[self.user_1, self.user_2],
+        )
+        self.team_2 = self.create_team(
+            organization=self.organization, slug="executives", members=[self.user_2]
+        )
+        self.project_1 = self.create_project(
+            organization=self.organization, teams=[self.team_1, self.team_2], slug="final-cover"
+        )
+        self.project_2 = self.create_project(
+            organization=self.organization, teams=[self.team_1], slug="number-25"
+        )
+        self.code_mapping_1 = self.create_code_mapping(project=self.project_1)
+        self.code_mapping_2 = self.create_code_mapping(project=self.project_2)
+        self.external_user = self.create_external_user(
+            user=self.user, external_name="@walter", integration=self.integration
+        )
+        self.external_team = self.create_external_team(
+            team=self.team_2, external_name="@life/exec", integration=self.integration
+        )
+        self.data_1 = {
+            "raw": "negatives/*  @life/exec @walter @hernando\nexec/* @life/exec\n",
+            "codeMappingId": self.code_mapping_1.id,
+        }
+        self.data_2 = {
+            "raw": "negatives/*  @life/exec @walter @hernando\nquintessence/* @walter @sean\n",
+            "codeMappingId": self.code_mapping_2.id,
+        }
+        self.login_as(user=self.user_1)
+
+    def test_no_codeowners(self):
+        response = self.get_success_response(self.organization.slug, status=status.HTTP_200_OK)
+        assert response.data == []
+
+    def test_simple(self):
+        """
+        Tests that all the ProjectCodeOwners are serialized in the response
+        """
+        code_owner_1 = self.create_codeowners(
+            self.project_1, self.code_mapping_1, raw=self.data_1["raw"]
+        )
+        code_owner_2 = self.create_codeowners(
+            self.project_2, self.code_mapping_2, raw=self.data_2["raw"]
+        )
+        response = self.get_success_response(self.organization.slug, status=status.HTTP_200_OK)
+        for code_owner in [code_owner_1, code_owner_2]:
+            assert (
+                serialize(
+                    code_owner,
+                    self.user_1,
+                    serializer=ProjectCodeOwnersSerializer(expand=["errors"]),
+                )
+                in response.data
+            )
+
+    def test_errors(self):
```

Comment:
nit: test name is slightly misleading. reads like you're testing error cases instead of verifying the error content of the response.

Reply:
Updated!